src/plugins/inputs/join.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+var Chan = require("../../models/chan");
+var _ = require("lodash");
+
+exports.commands = ["join"];
+
+exports.input = function(network, chan, cmd, args) {
+	var client = this;
+	if (args.length !== 0) {
+		var irc = network.irc;
+		irc.join(args[0], args[1]);
+	}
+
+	_.zip(args[0].split(/,/g), args[1].split(/,/g))
+		.map(function(chanPairs) {
+			return new Chan({
+				type: Chan.Type.CHANNEL,
+				name: chanPairs[0],
+				key: chanPairs[1],
+			});
+		})
+		.map(function(channel) {
+			network.channels.push(channel);
+			client.emit("join", {
+				network: network.id,
+				chan: channel
+			});
+		});
+
+	this.save();
```

Comment:
Yeah, I can see the code in `inputs/join.js` , but I'm kinda confused about why it isn't saving the channel name at least?